BuildScript.Nuke/GenerateSbom/SolutionSbomGenerator.cs

```diff
@@ -116,25 +118,56 @@ public void Generate (Tool cycloneDxTool, IReadOnlyCollection<ProjectInfo> proje
 
       _outputFile.Parent.CreateDirectory();
 
-      File.Copy(_workingDirectory / cleanedSbomFileName, _outputFile, overwrite: true);
+      //The sbom cleaner only works with xml files, so we need to convert the sbom if we want a different format.
+      var outputSbomFileToCopy = _workingDirectory / cleanedSbomFileName;
+      if (_outputFile.Extension.EndsWith("json"))
+      {
+        outputSbomFileToCopy = DuplicateAsJsonSbom(outputSbomFileToCopy);
+      }
+
+      File.Copy(outputSbomFileToCopy, _outputFile, overwrite: true);
 
       Log.Information($"Cleaned sbom has been copied to '{_outputFile}'.");
     }
   }
 
+  private AbsolutePath DuplicateAsJsonSbom (AbsolutePath outputSbomFileToCopy)
+  {
+    using var cleanedBomStream = File.OpenRead(outputSbomFileToCopy);
+    var finalBom = XMLSerializer.Deserialize(cleanedBomStream);
+
+    var outputSbomJson = _workingDirectory / "output-sbom.json";
```

Comment:
It feels strange to me that we get the input file via parameter but the output file is calculated from field state. I would suggest introducing a parameter for `outputSbomJson` and then we should be able to make this method `static` as well.